### DISCUSSION TITLE
CSS and other files not exporting

### DISCUSSION THREAD

**Discussion by truckker85** [webstudio-is/webstudio-community#192] [General]
I recently created my first website using webstudio however one of my pages called contact us is not getting exported onto my computer with css and other files but only html. The same page however looks as desired in the webstudio tool and with publishing as well.

**Comment by kof** [comment]
Is it using export UI and downloading from webstudio? or using CLI?

**Comment by truckker85** [comment]
I'm using export UI and downloading from webstudio into my local drive and then upload it to cpanel.

**Reply by kof** [reply]
<img width="213" alt="image" src="https://github.com/user-attachments/assets/50984da9-c81b-4151-bdfb-7dcfafa8c07f">

just tested, I see the file, is this a problem on a particular project of yours? have you tried a fresh one?

**Reply by truckker85** [reply]
My static folder doens't contain this file after I export and extract the file.

**Reply by truckker85** [reply]
or I should say it doesn't have .css extension in it's name, does that matter?

<img width="642" alt="css extension" src="https://github.com/user-attachments/assets/13c88ad6-3df3-4189-a955-38789a7aaa92">

**Comment by truckker85** [comment]
Yes I'm trying to get my website and the only page that I'm unable to complete the "contact us" page, see the difference of the page on webstudio vs actual website www.permitfinder.ca

<img width="949" alt="Contact us in the tool" src="https://github.com/user-attachments/assets/0f08d2d1-e72c-4177-be0a-d9368566c59b">
<img width="326" alt="Contact us on the website" src="https://github.com/user-attachments/assets/15c6f99d-91a2-4b52-a2f2-94667c4a1889">

**Reply by truckker85** [reply]
The project looks ok on the subdomain but I'm not able to get the same export on my local desktop??
https://permitfinder-mq27m.wstd.io/contact-us-1

Notice all other files are missing after exporting it to my local drive

<img width="838" alt="CSS and other files are missing" src="https://github.com/user-attachments/assets/87042ab1-4eff-4a6e-ab8d-2560618f8742">

**Reply by kof** [reply]
Have you tried like suggested an empty blank site? to see if this has anything to do with your specific changes?

**Reply by truckker85** [reply]
Not sure how I'd play around that, I could try something else. Wordpress?? Thanks for your help here !!

**Reply by kof** [reply]
The file is def. there. You are probably just trying to open the html file from the file system instead of starting a local http server or uploading it as is to a static hosting provider

**Comment by kof** [comment]
https://docs.webstudio.is/university/self-hosting#local
